types/led_state.yaml

```diff
@@ -0,0 +1,9 @@
+description: Possible LED states
+types:
+  LedState:
+    description: Current LED state
+    type: string
+    enum:
```

Comment:
wouldn't it not be more flexible to just send rgb? so we can have any color we want?

Reply:
That's a very valid point, let me rewrite the code